source/process/sg_grammar-spelling-mechanics.rst

```diff
@@ -132,17 +135,33 @@ Create numbered lists and procedure steps using arabic numerals for the top-leve
 
 1. This is the first step.
 2. This is the second step.
-  
+
   a. This is a substep.
   b. This is another substep.
-  
+
 3. This is the third step.
 
-Document Linking 
+Name-value Groups
+-----------------
+
+Use a name-value group instead of a hand-created list.
+
+A name-value group is typically a group of terms and their corresponding definitions, but can also be questions and answers, topics and values, or other name-value groups. In HTML output, a name-value group is represented as a definition list.
```

Comment:
The key advantage here is the intending. I think we should mention that

And maybe mention it's use in troubleshooting docs etc?